### PR TITLE
Merge server and client

### PR DESCRIPTION
**Pull request by modmuss50**
Do not merge just yet

### CODE REVIEW

File: patches/net/minecraft/client/multiplayer/PlayerControllerMP.java.patch [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-+            if(false)
-             if(this.currentGameType.isCreative()) {
-                 this.connection.sendPacket(new CPacketPlayerDigging(CPacketPlayerDigging.a.START_DESTROY_BLOCK, var1, var2));
-                 clickBlockCreative(this.mc, this, var1, var2);
-@@ -173,11 +175,13 @@
+@@ -130,7 +131,7 @@
+                 }
+ 
+                 this.currentBlock = new BlockPos(this.currentBlock.getX(), -1, this.currentBlock.getZ());
+-                if(!this.currentGameType.isCreative()) {
```
Comment (iLexiconn) [inline patches/net/minecraft/client/multiplayer/PlayerControllerMP.java.patch]: add this line back, just add the if(false) above it

**Comment by modmuss50** [comment]
Ok, should be good to merge

**Comment by iLexiconn** [comment]
You didn't add the `if(false)` back

**Comment by modmuss50** [comment]
I did, https://github.com/OpenModLoader/OpenModLoader/pull/38/commits/a256990973a83a4ee91328bcb1dd14cad16af9bb

**Comment by iLexiconn** [comment]
You removed it

**Comment by modmuss50** [comment]
well then It was fine before then?

**Comment by iLexiconn** [comment]
No, in the first commit you replaced a line with `if(false)`. I asked you to keep that line but add `if(false)` before that line instead.

**Pull request merged**